assets/css/styles.scss

```diff
@@ -249,11 +249,11 @@ h1, h2, h3, h4, h5, h6 {
   border-bottom: 1px solid darken(#ededed, 10%);
 }
 
-.btn-payment > .fa-spinner {
+.btn > .fa-spinner {
```

Comment:
if i am not mistaken this break the spinner in the cart button when you add products on the restaurant page (the button is empty)

Reply:
wow, nice catch. I'll fix it